src/pages/Selector/container.jsx

```diff
@@ -1,27 +1,58 @@
 import React from 'react';
+import { PropTypes as PT } from 'prop-types';
+import { compose } from 'redux';
+import { connect } from 'react-redux';
 import { fetchUsers, fetchTemplates } from 'Services';
+import { getEmployeeCV } from 'Reducers/selected-options';
 
 import { transformEmployees, transformTemplates } from './transform';
 
 const SelectorContainer = Wrapped =>
   class extends React.Component {
-    state = { employees: [], templates: [] };
+    static propTypes = {
+      employeeCV: PT.shape({ id: PT.number, label: PT.string }).isRequired,
+    };
+
+    state = { employees: [], templates: [], employeeCV: [], modalIsOpen: false };
 
     componentDidMount() {
       fetchUsers().then(employees => this.setState({ employees }));
       fetchTemplates().then(templates => this.setState({ templates }));
     }
 
+    componentDidUpdate(prevProps) {
+      const { employeeCV } = this.props;
+      if (prevProps.employeeCV !== employeeCV) {
+        this.setState({ employeeCV, modalIsOpen: true });
+      }
+    }
+
+    handleCancelModal = () => {
```

Comment:
`handleCancelModal = () => this.setState({ modalIsOpen: false });` small thing but you could write it like this.